src/web/components/Core/StatusPopup.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import { StatusPopup } from './StatusPopup';
+
+export default {
+  title: 'Shared Components/StatusPopup',
+  component: StatusPopup,
+} as ComponentMeta<typeof StatusPopup>;
+
+const Template: ComponentStory<typeof StatusPopup> = (args) => <StatusPopup {...args} />;
+
+export const Success = Template.bind({});
+Success.args = {
+  message: 'Success! Your action was successful.',
```

Comment:
Do we have a place to keep all the strings ? 
(might be helpful if we later want to do translations etc)